mcfly/modelgen.py

```diff
@@ -51,9 +47,9 @@ def generate_models(
     metrics : list
         Metrics to calculate on the validation set.
         See https://keras.io/metrics/ for possible values.
-    model_type : str, optional
-        Type of model to build: 'CNN' or 'DeepConvLSTM'.
-        Default option None generates both models.
+    model_types : str, list, optional
```

Comment:
it can only be a list, a single str does not work
```suggestion
    model_types : list, optional
```